packages/remix-react/__tests__/setup.ts

```diff
@@ -1,3 +1,8 @@
-import { installGlobals } from "@remix-run/node";
+const JSDOMFormData = global.FormData;
+global.TextDecoder = require("util").TextDecoder;
+global.TextEncoder = require("util").TextEncoder;
+global.ReadableStream = require("stream/web").ReadableStream;
+global.WritableStream = require("stream/web").WritableStream;
 
-installGlobals();
+require("@remix-run/node").installGlobals();
```

Comment:
It seems like `installGlobals` should also install `TextDecoder`, `TextEncoder`, `ReadableStream`, and `WritableStream`, no?

Reply:
This is a JSDom issue here.